README.rst

```diff
@@ -70,7 +70,7 @@ NI-DCPower Python API Status
 +-------------------------------+--------------------------+
 | NI-DCPower (nidcpower)        |                          |
 +===============================+==========================+
-| Driver Version Tested Against | 21.0.0                   |
+| Driver Version Tested Against | 21.3.0                   |
```

Comment:
It seems like these changes belong to a different PR.
The old nimi-bot is still online and hasn't been updated.
@sbethur what do you think, should we do upgrade old nimi-bot first and make the "Tested Against" changes along with that in own PR?

Reply:
@marcoskirsch  old nimi-bot has been deleted, there was some confusion. So it is little more effort is required to test against old nimi-bot

Reply:
@SukruthGrandhiNI, Marcos is talking about "Jenkins nimi-bot", which I just triggered for this PR.

I agree that we should update the "Driver Version Tested Against" in a separate PR.

I can update "Jenkins nimi-bot" but "IaC nimi-bot" is very close to being ready (on both platforms) and I don't mind waiting for it to be done before updating the Readme. It's ok to under-report the tested versions for a few days.